ui/src/views/offering/AddComputeOffering.vue

```diff
@@ -340,7 +340,7 @@
         </a-form-item>
         <a-form-item name="computeonly" ref="computeonly">
           <template #label>
-            {{ $t('label.computeonly.offering') }}
+            <tooltip-label :title="$t('label.computeonly.offering')" tooltip="Option to specify root disk related information in the compute offering or to link a disk offering to the compute offering"/>
```

Comment:
@nvazquez can you add the tooltip message to en.json file?

Reply:
I think, most of the tooltip descriptions are from the API params, and are not translated. In that case, this is in consistent with the other tooltips.

Reply:
Sorry @sureshanaparti have missed your comment - it is already fixed